File: src/main/java/me/waiaf/skytizencore/listeners/PlayerFirstJoin.java
package me.waiaf.skytizencore.listeners;

import me.waiaf.skytizencore.ItemManager;
import org.bukkit.entity.Player;
import org.bukkit.event.EventHandler;
import org.bukkit.event.Listener;
import org.bukkit.event.player.PlayerJoinEvent;

public class PlayerFirstJoin implements Listener {

    @EventHandler
    public void playerFirstJoinListener(PlayerJoinEvent event){

        Player player = event.getPlayer();

        if (player.hasPlayedBefore()) return;

        player.getInventory().setItem(8, ItemManager.menuItem);

    }
}

File: src/main/java/me/waiaf/skytizencore/listeners/PlayerFirstJoin.kt
package me.waiaf.skytizencore.listeners

import me.waiaf.skytizencore.ItemManager
import org.bukkit.event.EventHandler
import org.bukkit.event.Listener
import org.bukkit.event.player.PlayerJoinEvent

class PlayerFirstJoin : Listener {
    @EventHandler
    fun playerFirstJoinListener(event: PlayerJoinEvent) {
        val player = event.player
        if (player.hasPlayedBefore()) return
        player.inventory.setItem(8, ItemManager.menuItem)
    }
}